tests/conftest.py

```diff
@@ -67,6 +70,31 @@ def _factory(**kwargs):
     return _factory
 
 
+def _simple_factory(cls: type, default_kwargs: Union[dict, Callable[[dict], dict], None] = None):
+    def _factory(**kwargs):
+        kwargs = _common_kwargs(kwargs)
+        defaults = default_kwargs
+        if defaults:
+            if callable(defaults):
+                defaults = defaults(kwargs)
+            kwargs = {**defaults, **kwargs}
+        # It's very easy to accidentally set "default" to a raw value rather than a Value in our test
+        # code, which is never valid but mypy can't catch it for us. So we'll transform it here.
+        default_value = kwargs.get("default")
```

Comment:
This is the "inconsistent types in test code" issue that I mentioned [here](https://github.com/openapi-generators/openapi-python-client/pull/1096#discussion_r1742704815). I don't like the workaround, I'd rather require the tests to respect the right data types if they're going to call the constructors directly (which is what all of the property factories currently do), but I think that that deserves a separate PR for general cleanup of Value-related stuff.

Reply:
(Note: I believe it is possible to put type hints on pytest fixtures that would prevent that kind of problem, but currently it looks like mypy is disabled for all of the test code and it'd take quite a bit of cleanup to get it to pass. I think it's also worth considering whether the tests should even be constructing instances in this way, since that makes it easy to construct things in bad states that wouldn't really exist; I feel like maybe the fixtures should be using the `build` methods? But that'd require a lot of test code changes too.)